package.json

```diff
@@ -172,6 +172,7 @@
 		"nanoid": "^3.3.4",
 		"nest-winston": "^1.8.0",
 		"nestjs-console": "^8.0.0",
+		"node-html-parser": "^5.4.2-0",
```

Comment:
I dont like to add a additional parser. You are sure that nothing of the existing packages do the job you need?

Reply:
I delete the additional parser and rewrite the code for the existing 'sanitize-html' library. Pushed in last commit.